spec/features/hq/create_admin_users.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.feature 'Creating Admin Users', :type => :feature do
+
+  scenario 'After creation new Admin Users should be able to log in' do
+    i_sign_in_as_admin
+    visit new_hq_admin_user_path
+    fill_in 'Email', with: 'newadmin.user@example.com'
+    fill_in 'Password', with: 'Password'
+    fill_in 'Password confirmation', with: 'Password'
+    click_button 'Create Admin user'
```

Comment:
My preference would be to leave these 4 lines here I think. 
What we're testing is that our app is integrated with Devise correctly so that the back-end processing of this form works as expected, so it's particularly related to the completion of the fields on this form. Hence I think having the detail is appropriate.
Also, I don't think that a method such as `i_create_an_admin_user_with_credentials(email, pass)` would be reused in a different context. (You'd just use a factory and sign in instead).
What do you reckon?